src/frontend/components/Chat/SharedChatComponents/ChatAvatar/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from 'grommet';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+const messages = defineMessages({
+  avatarTitle: {
+    defaultMessage: "The user's avatar",
+    description: 'A title describing the avatar icon purpose.',
+    id: 'component.ChatAvatar.avatarTitle',
+  },
+});
+
+const AvatarBox = styled(Box)`
+  // I set up a minWidth with here because otherwise avatar width is crushed by the margin of the component next to the avatar
```

Comment:
we don't use I in the comment
```suggestion`
  // minWidth is set otherwise avatar width is crushed by the margin of the component next to the avatar
```
please order properties in alphabetical order

Reply:
I changed the comment and removed the 'personnal' style.
I also re-ordered all components and styled-components properties by alphabetical order.
see commit [370064f2627cde1b08aaddeb0ef7efa5fa1377f4](https://github.com/openfun/marsha/pull/1247/commits/370064f2627cde1b08aaddeb0ef7efa5fa1377f4)